tool/matomo/README.md

```diff
@@ -0,0 +1,22 @@
+# Matomo (formerly Piwik)
```

Comment:
I've also realized this is a new readme file. Even though this is specific to the tool_matomo I think this should be a section under the main readme purely to make it easier to find when casually browsing github so you don't have to go dig. We could just cross link the two readmes but this chunk of help is small enough that we can just embed it directly.

In theory if we added this same support to GA and other tools they could all be inline in the main readme

Reply:
Have moved this to the parent plugin README